src/pdsfile/holdings_maintenance/_linkshelf_common.py

```
@@ -0,0 +1,712 @@
+            if key in shelfdict:
+                dirinfo = dirdict[key]
+                shelfinfo = shelfdict[key]
+
+                if isinstance(dirinfo, list):
+                    dirinfo.sort()
+
+                if isinstance(shelfinfo, list):
+                    shelfinfo.sort()
+
+                if dirinfo != shelfinfo:
+                    logger.error('Link target mismatch', key)
```

Ruff (0.16.1)

[error] Too many arguments for `logging` format string

(PLE1205)